### ISSUE TITLE
Button: hover/focus does not work with aria-label and icon variant

### ISSUE THREAD

**Issue by PrashantAshok** [eBay/skin#1480]
`ebay-button` when it has variant="icon" and aria-label - renders an wrapper span over the svg which is causing the hover/focus state to not work. 

https://github.com/eBay/skin/blob/master/src/less/icon-button/base/icon-button.less#L38-L39
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-button/index.marko#L74

Maybe the code should be less specific
```
&:focus svg,
&:hover svg {
  .customFillProperty(icon-button-icon-hover-foreground-color);
}
```

**Comment by agliga** [comment]
This will be fixed on the ebayui-side

**Comment by ianmcburnie** [comment]
Closing as not an issue (on skin side)
